packages/pmml-editor/src/editor/components/Outputs/molecules/OutputFieldEditRow.tsx

```diff
@@ -126,7 +138,11 @@ const OutputFieldEditRow = (props: OutputFieldEditRowProps) => {
       eventTypes: ["click"]
     }
   );
-
+  const { service } = useValidationService();
```

Comment:
Since `useValidationService` has a `service` property, maybe this Custom Hook should become `useValidation` and you could change this to

```suggestion
  const validationService = useValidation().service;
```

WDYT?

Reply:
This has all changed in my PR #406.. 

It is now `ValidationRegistry` and the `string` literal paths changed to the builder I added.

I propose this PR be merged (once you're OK with it) and my #406 will then be updated to include the work in here.

I pasted a dependency chain of PMML PRs on `tooling` on Zulip and updated PR labels in GitHub.

Reply:
:+1: